01/EnumExample.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>

```

Comment:
You don't need to have Example postfix on your file names. Also, let's make a separate folder for each example (in the later lectures, we will have multiple files per example... so let's make it consistent)
ex>
```
Enum
    - Enum.c
GlobalVariable
    - GlobalVariable.c
```